tests/vmi_configuration_test.go

```diff
@@ -579,6 +579,32 @@ var _ = Describe("Configurations", func() {
 				Expect(err).ToNot(HaveOccurred(), "should report number of usb")
 			})
 
+			It("should start the VMI with usb controller when input device doesn't have bus", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
+				vmi.Spec.Domain.Devices.Inputs = []v1.Input{
+					{
+						Name: "tablet0",
+						Type: "tablet",
```

Comment:
should not conversion fail before the start because of not supported `tablet` type [link](https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-launcher/virtwrap/api/converter.go#L455)?

Reply:
ah my bad, it's correct. Ignore the comment above.